snap/implicit.go

```diff
@@ -34,11 +34,13 @@ var implicitSlots = []string{
 	"hardware-observe",
 	"locale-control",
 	"log-observe",
+	"mir",
```

Comment:
agreed, simply removed until we know more wrt personal and the future, will work through the classic case next